### ISSUE TITLE
Add Seshat capitals map view

### ISSUE THREAD

**Issue by edwardchalstrey1** [edwardchalstrey1/seshat#11]
- [x] Plot capital coordinates on map
- [ ] Click on them to get to Seshat pages

**Comment by edwardchalstrey1** [comment]
Get capitals for a polity using the general_polity_capital table e.g.

```
select cp.name, cp.id, cp.start_year, cp.end_year, gpc.capital, cc.latitude, cc.longitude
from core_polity as cp, general_polity_capital as gpc, core_capital as cc
where cp.id = gpc.polity_id
and cc.name = gpc.capital
and cp.name='JpAsuka';
```

**Comment by edwardchalstrey1** [comment]
Closed by #66
